lib/content_security_policy.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+require_dependency 'global_path'
+
+class ContentSecurityPolicy
+  include GlobalPath
+
+  class Middleware
+    WHITELISTED_PATHS = %w(
+      /logs
```

Comment:
Logster still has [quite some inline JS](https://github.com/discourse/logster/blob/0b4b42fe815b333f56bf7daa804b1aa2fb0642cc/lib/logster/middleware/viewer.rb#L204-L248).

Reply:
Ouch ... can you work with @OsamaSayegh on getting that removed asap?